app-core/src/main/java/net/spals/appbuilder/app/core/WorkerAppBuilder.java

```diff
@@ -51,6 +51,30 @@
      */
     WorkerAppBuilder<A> disableErrorOnServiceLeaks();
 
+    /**
+     * Allow bindings in standard and custom Guice
+     * {@link Module}s to override each other.
+     *
```

Comment:
FWIW you should place <p> between paragraphs so it appears that way when you view the javadoc. You can set up intellij to inject that for you.